src/zh/zaimanhua/src/eu/kanade/tachiyomi/extension/zh/zaimanhua/ZaimanhuaDto.kt

```diff
@@ -170,3 +179,46 @@ data class ImageRetryParamsDto(
     val url: String,
     val index: Int,
 )
+
+@Serializable
+data class CanReadDto(
```

Comment:
```suggestion
class CanReadDto(
```

Same for other classes.